src/main/java/edu/phema/elm_to_omop/repository/ObjectMapperContextResolver.java

```diff
@@ -0,0 +1,22 @@
+package edu.phema.elm_to_omop.repository;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import javax.ws.rs.ext.ContextResolver;
+import javax.ws.rs.ext.Provider;
+
+@Provider
+public class ObjectMapperContextResolver implements ContextResolver<ObjectMapper> {
+    private final ObjectMapper mapper;
+
+    public ObjectMapperContextResolver() {
+        mapper = new ObjectMapper();
+        mapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
```

Comment:
This is just to avoid posting a bunch of `null` values to the WebAPI. It's not strictly necessary, but it seemed nicer to me.